opendevin/server/cert/README.md

```diff
@@ -0,0 +1,16 @@
+# cert
+Attention! Self-signed certificates only for development purposes. If you need to test the TLS for `HTTP` and `WS`, just run the following command:
```

Comment:
Uploading your personal public and private keys for development purposes is not a secure practice, even if others will be using the repository to set up their own instances of the application. Security best practices should be adhered to at all times to prevent unauthorized access or impersonation. A better approach is to include a script in the repository, or detailed instructions, guiding users through the process of generating their own set of SSL/TLS certificates. This will ensure that each individual's development setup remains secure and that the keys remain unique to their environment.

Reply:
While I don't think there's much risk in checking this in, I agree with Umpire that it makes sense to just tell the dev how to generate certs. I don't think we'll need to test this much now that it's done.

Reply:
yep, it makes sense. @Umpire2018 thanks for your advice.